src/PerfViewExtensions/GlobalSrc/Global.csproj

```diff
@@ -11,24 +11,6 @@
     <AssemblyName>Global</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <!-- Determine if we are building 'next to' the main PerfView sources -->
```

Comment:
This funky logic has a purpose.   Global.csproj is actually mean to support adding extensions to PerfView.   In perfView go to Help -> Extending PerfView for the experience.   Bascially perfView can set up a SLN that allows you to build things that extend perfView.   This Global.csproj is the template for this (We build it here just just so we are less likely to break the experience).   In 'Normal' use this project refers to the PerfVIew.exe that is placed in the 'installed location', but at build time it refers to where the build put it.  

In short, unless you want to revamp that, leave this alone.   You should also test the Extention experience to insure we did not break it.